smoke_test/remote/README.md

```diff
@@ -0,0 +1,13 @@
+# Adding new remote examples
```

Comment:
Going to be honest, not sure I'm tracking with you. Is the issue that you don't like splitting into `remote` and `vendor` in the first place?

If so, I wasn't a fan originally either. However, having done this, I think it makes a lot of sense, given that one of those needs special bits added to the WORKSPACE file (though come to think of it, it might be possible to put in remote/BUILD).

Reply:
This readme, verbatim: `Adding new remote examples`

What I meant, was that I don't really _want_ people to be adding new examples -- at least of the form that have their own Cargo.toml and whatnot.  I'm under the impression that each one of these results in about three minutes of additional runtime.

I'm wondering if you have given any thought at all to a testing strategy that doesn't require adding whole new WORKSPACES and Cargo.tomls, or if you have any opinion about me compacting them at some later time. It sounds weird having spelled it out in this manner though...

Reply:
Couple of thoughts:

1. We want to have materialized examples so people can see what the results look like, given this is a new tool.

2. We're using smoke-test as a shortcut around building proper testing support in the first place. I think it would be wise to invest some time in a more robust test suite (preaching to the choir, I know).

3. What *actually* takes a long time is compiling projects after we've generated the files for them. In essence, we're using bazel to validate that our work is correct. If instead we had a smoke test that could validate a directory diff or something similar we could skip the bazel step; the interesting bits don't ever change *because of* bazel compilation.

I think 3 is the direction to go in the near term to get a smoke test (because testing real-world scenarios will always be good) that doesn't take forever. There are going to be significant issues because of changing version numbers breaking the diff, etc., but I think ultimately we want to separate "this will build" from "this has been built."